src/playermat/Playermat.ttslua

```diff
@@ -252,13 +252,7 @@ end
 -- searches the area around the draw deck and discard pile
 function searchDeckAndDiscardArea(filter)
   local pos = self.positionToWorld(DECK_DISCARD_AREA.center)
-  local scale = self.getScale()
-  local size = {
-    x = DECK_DISCARD_AREA.size.x * scale.x,
-    y = DECK_DISCARD_AREA.size.y,
-    z = DECK_DISCARD_AREA.size.z * scale.z
-  }
-  return searchArea(pos, size, filter)
+  return searchArea(pos, DECK_DISCARD_AREA.size * self.getScale(), filter)
```

Comment:
No functional change, just implemented Vector multiplication to make it more concise